src/test/java/reactor/ipc/netty/resources/DefaultPoolResourcesTest.java

```diff
@@ -114,4 +126,68 @@ public void disposeOnlyOnce() {
 		assertThat(closed.get()).as("pool closed only once").isEqualTo(1);
 	}
 
+	@Test
+	public void fixedPoolTwoAcquire()
+			throws ExecutionException, InterruptedException, IOException {
+		final ScheduledExecutorService service = Executors.newScheduledThreadPool(2);
+		int echoServerPort = SocketUtils.findAvailableTcpPort();
```

Comment:
FYI, this is not entirely reliable and can cause problems on CI servers, if some other process grabs the socket. We've seen this happen once in a while, causing bogus CI build failures. 

In my projects, where possible, we've changed tests to listen on port 0 (so the OS selects the port) and then query the test server for which port was actually selected.

Not critical but worth consideration.

Reply:
hard to do as the first part of the test checks what happens when the host is initially unreachable. crossing fingers for CI :(